packages/sanity/src/structure/comments/src/components/pte/comment-input/CommentInput.tsx

```diff
@@ -26,12 +26,12 @@ export interface CommentInputProps {
   mentionOptions: UserListWithPermissionsHookValue
   onBlur?: (e: React.FormEvent<HTMLDivElement>) => void
   onChange: (value: PortableTextBlock[]) => void
-  onDiscardCancel: () => void
-  onDiscardConfirm: () => void
+  onDiscardCancel?: () => void
```

Comment:
This changes will be reverted or refactored once we land the `FormBuilder` changes into tasks.